setup_docker.sh

```diff
@@ -1,5 +1,9 @@
 #kills the running process, and creates a new web-server to run
 docker kill "$(docker ps -q)"
+docker stop $(docker ps -a -q)
```

Comment:
Found those commands online to basically remove all images that lingered from before, so we start with a completely new docker machine everytime.